sabnzbd/postproc.py

```diff
@@ -574,6 +574,7 @@ def process_job(nzo: NzbObject) -> bool:
                     # Deobfuscate the filenames
                     logging.info("Running deobfuscate")
                     deobfuscate.deobfuscate(nzo, newfiles, nzo.final_name)
+                    deobfuscate.deobfuscate_subtitles(workdir_complete)
```

Comment:
I just can't be used like this.
For users that use the `folder/*` notation for their Category settings, all jobs will be unpacked to the same folder without sub-folders for each job. So the `workdir_complete` could contain 10 different movies and assosiated files.
And a lot of users have this, since it's the solution when Googling for "stop SABnzbd from creating job folders".

You will need to change `deobfuscate.deobfuscate` to update the `newfiles` list, and only use that list as your input for `deobfuscate.deobfuscate_subtitles`.

If you want, I can make that change (the updating of `newfiles` in `deobfuscate.deobfuscate`)?

Reply:
Yes, great if you can do that.

Reply:
```suggestion
                    deobfuscate.deobfuscate_subtitles(newfiles)
```
You didn't use the output of the previous one?